### PR TITLE
feat: new selectablecard component

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Feature

#### What is expected?

Creation of new component SelectableCard.

## Relevant logs and/or screenshots

https://user-images.githubusercontent.com/15812968/182827950-be9fcb75-044b-4264-80b6-262afc1bb246.mov

### CODE REVIEW

File: src/components/Radio/index.tsx [hunk 1]
```diff
@@ -139,34 +140,42 @@ const Radio = ({
...
-    htmlFor={`${name}-${value}`}
-    className={className}
-  >
-    <StyledRadio
-      type="radio"
-      aria-invalid={!!error}
-      aria-checked={checked}
+}: RadioProps) => {
+  const computedName = useMemo(() => {
+    if (!name) return getUUID('checkbox')
```
[the remark this answers is not in the document]
Comment (matthprost) [reply src/components/Radio/index.tsx:145]: @Exayy yes you're right I simply copied and paste from checkbox and forgot to change the inside value

File: src/components/SelectableCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import styled from '@emotion/styled'
+import React, { ChangeEventHandler, ReactNode } from 'react'
+import Checkbox from '../Checkbox'
+import Radio from '../Radio'
+
+const StyledElement = (element: typeof Radio) => styled(element, {
+  shouldForwardProp: prop => !['showTick'].includes(prop),
+})<{ showTick?: boolean }>`
```
[the remark this answers is not in the document]
Comment (matthprost) [reply src/components/SelectableCard/index.tsx:8]: @Exayy Forgot about the disabled one, I will add it thanks

File: src/components/SelectableCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  value: string | number
+  onChange: ChangeEventHandler<HTMLInputElement>
+  showTick?: boolean
+  type?: 'radio' | 'checkbox'
+  disabled?: boolean
+  checked?: boolean
+  className?: string
+}
+
+const SelectableCard = ({
```
Comment (philibea) [inline src/components/SelectableCard/index.tsx:55]: you should add isError or error props to be able to show visual error on this component and children when we will use it on a FormField

Comment (matthprost) [reply src/components/SelectableCard/index.tsx:55]: @philibea I will add a isError for the error style, for an error message a SelectableCardGroup will be required

**Review by philibea — APPROVED** [review]

**Review by Exayy — APPROVED** [review]

**Pull request merged**